requirements-test.txt

```diff
@@ -1 +1 @@
-numpy
+numpy<=1.23.0
```

Comment:
Why is this necessary?

Reply:
https://numpy.org/doc/stable/release/1.24.0-notes.html